csrank/discretechoice/feta_discrete_choice.py

```diff
@@ -202,6 +205,42 @@ def get_score_object(i):
         model.compile(loss=self.loss_function, optimizer=self.optimizer, metrics=self.metrics)
         return model
 
+    def _create_weighted_model(self, n_objects):
```

Comment:
This change is big enough to warrant a separate pull request.

Reply:
Can we just merge this change in this one and from the next one make different pull requests.